app/main/controllers/template/RTMediaNav.php

```diff
@@ -454,4 +456,95 @@ function set_privacy( $profile ) {
 
 		return $privacy;
 	}
+
+	/**
+	 * Check if media component has counts.
+	 *
+	 * @param bool   $count  Component feed has count.
```

Comment:
```suggestion
	 * @param bool   $has_count  True if the nav has a count attribute. False otherwise
```
Could we use an other variable name than count, since the variable has boolean value, `count` as a name is misleading.